lib/ai/tools/run-terminal-cmd.ts

```
@@ -53,14 +59,51 @@ In using these tools, adhere to the following guidelines:
+                    stdout: finalResult.stdout,
+                    stderr: finalResult.stderr,
+                  },
+                });
+              }
+            } catch (error) {
+              handler.cleanup();
+              if (!resolved) {
+                throw error;
+              }
+            }
+          });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)